app/models/conversation.rb

```diff
@@ -293,4 +291,9 @@ def mute_key
   def mute_period
     6.hours
   end
+
+  # creating db triggers
+  trigger.before(:insert).for_each(:row) do
+    "NEW.display_id := nextval('account_seq_' || NEW.account_id);"
```

Comment:
probably name the sequence as. `conv_dpid_seq_`  ?   
if we end up adding more display id sequences on like . `campaign_dp_id`